test/integration/camera_sensor.cc

```diff
@@ -40,10 +40,14 @@ unsigned char* img = NULL;
 unsigned char* img2 = NULL;
 unsigned char* img3 = NULL;
 unsigned char* img4 = NULL;
+unsigned char* img5 = NULL;
```

Comment:
Hey @kjeppesen1, it looks like c3f5cf7 didn't make any changes and 464adb1 reapplied some old changes (e.g., I removed my additions to this test, among other things). We could cherry-pick 4abe770 on top of 406b65e? I can probably do this tomorrow, as well as address @scpeters's feedback.